tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Service/Microsoft.Legal.MatterCenter.Repository/SharePointInfo/UsersDetails.cs

```diff
@@ -156,6 +158,44 @@ public bool CheckUserPresentInMatterCenter(string clientUrl, string email)
         }
 
         /// <summary>
+        /// This method will check whether the given login user is present in a 
+        /// sharepoint group or not
+        /// </summary>
+        /// <param name="client"></param>
+        /// <param name="groupName"></param>
+        /// <returns></returns>
+        public bool CheckUserPermissionsInGroup(Client client, string groupName)
```

Comment:
Added code to check whether the login user can create matter or not
Need to check whether login user is present in the sharepoint group or not
The application will throw the exception if the user does not belong to the group (Matter Provision Users) and he was having read permissions on the site collection and in the exception I am returning false

If the user is a site collection admin and if the user does not belong to the group, then the method will return false